gluoncv/auto/estimators/image_classification/image_classification.py

```diff
@@ -19,11 +19,12 @@
 from .... import nn
 from .... import loss
 from ..base_estimator import BaseEstimator, set_default
-from .utils import get_data_loader, get_data_rec, smooth
+from .utils import get_data_loader, get_data_rec, smooth, mixup_transform
 from .default import ImageClassificationCfg
 from ...data.dataset import ImageClassificationDataset
 from ..conf import _BEST_CHECKPOINT_FILE
 from ..utils import EarlyStopperOnPlateau
+from autogluon.core.constants import MULTICLASS, BINARY, REGRESSION
```

Comment:
move to inside `_fit` function

Reply:
These constants are used in multiple functions. Would it be better if they are defined here? @zhreshold